src/DropdownMenu.js

```diff
@@ -110,7 +110,7 @@ class DropdownMenu extends React.Component {
                     className,
                     bsPrefix,
                     show && 'show',
-                    alignRight && `${bsPrefix}-right`,
+                    alignEnd && `${bsPrefix}-right`,
```

Comment:
no extra intelligence needed here for left/right positioning?

Reply:
There is via popper, but this is an additional class, presumably to reset styles appropriately for that or the navbar where popper isn't used.